src/main/java/use_cases/collaborative_task_scheduling/scheduling_ct_use_case/ScheduleCTRequestModel.java

```diff
@@ -15,9 +18,9 @@ public class ScheduleCTRequestModel {
 
     private final String endTime;
 
-    private final StudentUser studentUser;
+    private final Object studentUser;
 
-    public ScheduleCTRequestModel(String taskName, String startTime, String endTime, StudentUser studentUser) {
+    public ScheduleCTRequestModel(String taskName, String startTime, String endTime, Object studentUser) {
```

Comment:
Again, could retrieve a StudentUser object here obtained from CurrentUser instead of taking it as a param